src/main/java/seedu/address/model/order/Price.java

```diff
@@ -11,7 +14,7 @@ public class Price {
 
     public static final String MESSAGE_CONSTRAINTS =
             "Price must be a positive number";
```

Comment:
Could you change this constraint to be more specific? such as "Price must be a positive number and can only include up to two decimal places."